rest-api/edit-conversation-image-title/server/server.js

```diff
@@ -0,0 +1,40 @@
+import express from 'express';
+import cors from 'cors';
+import bodyParser from 'body-parser';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const app = express();
+app.use(cors())
+app.use(bodyParser.json())
+
+async function editImageOrTitle(conversationId, imageURL, conversationTitle) {
+  
+  const data = conversationTitle === "" ? { photoUrl: imageURL } : imageURL === "" ? { subject: conversationTitle } : { subject: conversationTitle, photoUrl: imageURL };
```

Comment:
Can you break this into some if statements pls, it's impossible to follow what's going on here

Reply:
This still needs doing